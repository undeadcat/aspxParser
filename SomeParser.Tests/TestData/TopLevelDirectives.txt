<%@ Page Title="Деньги" Language="C#" MasterPageFile="~/Business/BusinessPage.Master" AutoEventWireup="true" CodeBehind="PaymentsList.aspx.cs" Inherits="Front.Business.Payments.List.PaymentsList" %>
<%@ Import Namespace="Kontur.Elba.Web.Infrastructure.Common.Mvc.Views" %>
<%@ Register TagPrefix="ib" Namespace="Front.Accounting" Assembly="Elba.Front" %>
<asp:Content ContentPlaceHolderID="PageContent" runat="server">
<div></div>
</asp:Content>
<asp:Content ContentPlaceHolderID="Footer" runat="server">
<div></div>
</asp:Content>