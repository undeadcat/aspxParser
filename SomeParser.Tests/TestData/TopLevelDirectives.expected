<AspNetXmlDocument>
Elements: 
    <TopLevelDirective>
    Name: 
        <CompositeIdentifier>
        Components: 
            Page
    Attributes: 
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                Title
        Value: 
            Деньги
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                Language
        Value: 
            C#
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                MasterPageFile
        Value: 
            ~/Business/BusinessPage.Master
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                AutoEventWireup
        Value: 
            true
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                CodeBehind
        Value: 
            PaymentsList.aspx.cs
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                Inherits
        Value: 
            Front.Business.Payments.List.PaymentsList
    <TopLevelDirective>
    Name: 
        <CompositeIdentifier>
        Components: 
            Import
    Attributes: 
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                Namespace
        Value: 
            Kontur.Elba.Web.Infrastructure.Common.Mvc.Views
    <TopLevelDirective>
    Name: 
        <CompositeIdentifier>
        Components: 
            Register
    Attributes: 
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                TagPrefix
        Value: 
            ib
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                Namespace
        Value: 
            Front.Accounting
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                Assembly
        Value: 
            Elba.Front
    <Tag>
    Name: 
        <CompositeIdentifier>
        Components: 
            asp
            Content
    Attributes: 
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                ContentPlaceHolderID
        Value: 
            PageContent
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                runat
        Value: 
            server
    Content: 
        <Tag>
        Name: 
            <CompositeIdentifier>
            Components: 
                div
        Attributes: 

        Content: 

    <Tag>
    Name: 
        <CompositeIdentifier>
        Components: 
            asp
            Content
    Attributes: 
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                ContentPlaceHolderID
        Value: 
            Footer
        <Attribute>
        Name: 
            <CompositeIdentifier>
            Components: 
                runat
        Value: 
            server
    Content: 
        <Tag>
        Name: 
            <CompositeIdentifier>
            Components: 
                div
        Attributes: 

        Content: 
